app/models/concerns/form526_claim_fast_tracking_concern.rb

```diff
@@ -277,5 +283,15 @@ def conditionally_notify_mas
                          Settings.rrd.mas_tracking.recipients)
     StatsD.increment("#{RRD_STATSD_KEY_PREFIX}.notify_mas.failure")
   end
+
+  def conditionally_merge_ep
+    pending_claim_id = read_metadata(:ep_merge_pending_claim_id)
+    return unless pending_claim_id.present? && Flipper.enabled?(:disability_526_ep_merge_api)
```

Comment:
Do we know if we are going to enable this for a subset of users like with the max-cfi effort?

Reply:
Currently there's no intention to do an A/B test rollout, since we're not trying to affect or measure user behavior. If we do want to do a gradual rollout, it would be much easier to do it on the VRO API side.